### PR TITLE
[PLA-1423] Remove account field replaced by signingAccount

### PR DESCRIPTION
**Pull request by leonardocustodio**

### CODE REVIEW

File: src/GraphQL/Mutations/CreateListingMutation.php [hunk 1]
```diff
@@ -64,10 +63,6 @@ public function type(): Type
     public function args(): array
     {
         return [
-            'account' => [
```
Comment (enjinabner) [inline src/GraphQL/Mutations/CreateListingMutation.php:67]: `signingAccount` field by default is optional, should we make it required like `account`

File: tests/Feature/GraphQL/Resources/CreateListing.graphql [hunk 2]
```diff
@@ -1,5 +1,4 @@
 mutation CreateListing(
-  $account: String!
```
Comment (enjinabner) [inline tests/Feature/GraphQL/Resources/CreateListing.graphql:2]: I think we should add `signingAccount` add add tests

**Review by enjinabner — CHANGES_REQUESTED** [review]

File: src/GraphQL/Mutations/CreateListingMutation.php (hunk 1 above)
Comment (leonardocustodio) [reply src/GraphQL/Mutations/CreateListingMutation.php:67]: Why is an account required? I'm not even sure why `account` did exist maybe I missed something, do you know?

Comment (v16Studios) [reply src/GraphQL/Mutations/CreateListingMutation.php:67]: Right, `account` doesn't appear to be actually used in the mutation or encodable data etc.  We should remove that and use the optional `signingAccount` flow to keep it consistent with all the other mutations 👍🏻

**Review by v16Studios — APPROVED** [review]

**Pull request merged**